### ISSUE TITLE
Unable to install in a new workspace - needs updated slack oauth methods

### ISSUE THREAD

**Issue by clottman** [WeAllJS/wheelie-slack-app#25]
I'm trying to set this up for the Web-A11y Slack, and ran into an error when attempting to authenticate/install through Slack. The error was `oauth_authorization_url_mismatch` which is a [slack oauth error code](https://api.slack.com/methods/oauth.v2.access). 

Probably the answer is to update the `slack` npm package that seems to be handling authentication. I am going to do that, and hopefully make a PR in the next few days with that/whatever else is required, plus a couple small documentation updates to account for changes in Slack's App Builder UI.

**Comment by clottman** [comment]
This is going to be more work than I expected. The `slack` package hasn't been updated in a year or more, and doesn't support Slack's oauth.v2 method. It looks like slack's auth methods (and bot building recommendations in general) have changed pretty substantially. Gonna try to muddle through anyways.

**Comment by shushugah** [comment]
this would be deeply appreciated!
